app.rb

```diff
@@ -13,8 +13,13 @@ class Uploader < Sinatra::Base
 
     get '/healthcheck' do
       {
+        service_status: 'OK',
```

Comment:
I considered it when I first moved it over, but wasn't sure it was needed.  Will do.